src/main/java/tools/jackson/databind/DeserializationFeature.java

```diff
@@ -110,12 +110,12 @@ public enum DeserializationFeature implements ConfigFeature
      * This setting only takes effect after all other handling
      * methods for unknown properties have been tried, and
      * property remains unhandled.
+     * Enabling this feature means that a {@link DatabindException}
+     * will be thrown if an unknown property is encountered.
      *<p>
-     * Feature is enabled by default (meaning that a
-     * {@link DatabindException} will be thrown if an unknown property
-     * is encountered).
+     * Feature is disabled by default.
```

Comment:
In this case, we probably do want to mention that the default state was different in Jackson 2.x.